framework/helpers/BaseStringHelper.php

```diff
@@ -374,6 +374,7 @@ public static function floatToString($number)
      * @param array $options options for matching. Valid options are:
      *
      * - caseSensitive: bool, whether pattern should be case sensitive. Defaults to `true`.
+     * - escape: bool, whether backslash escaping is enabled. Defaults to `true`.
```

Comment:
Why disabling it?

Reply:
This option matches `FNM_NOESCAPE` flag of `fnmatch()`.
Its disabling may be useful while processing patterns for strings with backslashes such as file path for Windows. So you can write `'begin\*\end'` instead of `'begin\\\\*\\\\end'`.